039-AKSForEnterprise/README.md

```diff
@@ -0,0 +1,43 @@
+# What The Hack - AKS Enterprise-Grade
+
+## Introduction
+
+This Hack has an objective studying in depth the integration surface between AKS and different Azure services. Some are optional components (for example monitoring), but others are required to successfully operate Kubernetes clusters on Azure (such as networking or storage).

```

Comment:
Let's change this paragraph to  make it a little bit clearer:

The objective of this hack is to dive deeply into all aspects of integration between AKS and various Azure services. We have included challenges for many components that are required to operate a Kubernetes cluster on Azure (eg: networking, storage, etc) as well as some optional ones (eg: monitoring).

Reply:
Done!